models/fates/R/met2model.FATES.R

```diff
@@ -61,103 +67,119 @@ met2model.FATES <- function(in.path, in.prefix, outfolder, start_date, end_date,
         ## specific_humidity
-        ncout <- insert(ncout, "SHUM", "kg/kg", SHUM)
-        
+        ncout_tem <- insert(ncout_tem, "QBOT", "kg/kg", QBOT[tsel], dim)
         ## air_temperature
-        ncout <- insert(ncout, "TBOT", "K", TBOT)
-        
+        ncout_tem <- insert(ncout_tem, "TBOT", "K", TBOT[tsel], dim)
         ## eastward_wind & northward_wind
-        ncout <- insert(ncout, "WIND", "m/s", WIND)
+        ncout_tem <- insert(ncout_tem, "WIND", "m/s", WIND[tsel], dim)
+        ncdf4::nc_close(ncout_tem)
         
-        ncdf4::nc_close(ncout)
-        
-        #   ncvar_rename(ncfile,varid="LONGXY")
-        #   ncvar_rename(ncfile,varid="LATIXY")
-        #   #     
-        #   #     double EDGEW(scalar) ;
-        #   #     EDGEW:long_name = "western edge in atmospheric data" ;
-        #   #     EDGEW:units = "degrees E" ;
-        #   EDGEW = ncvar_rename(ncfile,"EDGEW","EDGEW")
-        #   
-        #   #     double EDGEE(scalar) ;
-        #   #     EDGEE:long_name = "eastern edge in atmospheric data" ;
-        #   #     EDGEE:units = "degrees E" ;
-        #   EDGEE = ncvar_rename(ncfile,"EDGEE","EDGEE")
-        #   
-        #   #     double EDGES(scalar) ;
-        #   #     EDGES:long_name = "southern edge in atmospheric data" ;
-        #   #     EDGES:units = "degrees N" ;
-        #   EDGES = ncvar_rename(ncfile,"EDGES","EDGES") 
-        #   #     
-        #   #     double EDGEN(scalar) ;
-        #   #     EDGEN:long_name = "northern edge in atmospheric data" ;
-        #   #     EDGEN:units = "degrees N" ;
-        #   EDGEN = ncvar_rename(ncfile,"EDGEN","EDGEN")
       }
-      
-      ncdf4::nc_close(nc)
-    }  ## end file exists
-  }  ### end loop over met files
+      ncdf4::nc_close(nc) #PEcAn input file
+    }  ## end input file exists
+  }  ### end year loop over met files
   
-  PEcAn.logger::logger.info("Done with met2model.FATES")
+  #PEcAn.logger::logger.info("Done with met2model.FATES")
   
-  return(data.frame(file = paste0(outfolder, "/"), 
-                    host = c(PEcAn.remote::fqdn()), 
-                    mimetype = c("application/x-netcdf"), 
-                    formatname = c("CLM met"), 
-                    startdate = c(start_date), 
-                    enddate = c(end_date), 
-                    dbfile.name = "", 
-                    stringsAsFactors = FALSE))
+  return(LATIXY)
+ # (data.frame(file = paste0(outfolder, "/"), 
+ #                   host = c(PEcAn.remote::fqdn()), 
+ #                   mimetype = c("application/x-netcdf"), 
+ #                  formatname = c("CLM met"), 
+ #                   startdate = c(start_date), 
+ #                   enddate = c(end_date), 
+ #                   dbfile.name = "", 
+ #                   stringsAsFactors = FALSE))
 } # met2model.FATES
+
+met2model.FATES(in.path='/Users/mac/Documents/pecan/models/fates/R/FI-Hyy',in.prefix='/FLX_FI-Hyy_FLUXNET2015_FULLSET_HH_1996-2018_beta-3',
```

Comment:
These lines seem to be remnants from your development, please remove them as they are your local paths. You can document the function arguments and write tests instead